tests/utimensat-0.c

```diff
@@ -1,3 +1,6 @@
+#pragma GCC push_options
```

Comment:
We could check for the right GCC versions but this isn't a new option either, and not sure we care enough for the testsuite.